### PR TITLE
Use v3.10.0-alpha.0 repos for origin 3.10

### PR DESCRIPTION
**Pull request by elad661**
We can't use master now that origin has branched.

### CODE REVIEW

File: deployer.sh [hunk 1]
```diff
@@ -163,7 +163,7 @@ RETRCODE=0
 
 function install_repo() {
     IP=${1}
-    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "curl  https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/master/origin.repo > /etc/yum.repos.d/origin-master.repo; yum update -y"
+    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "curl  https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/v3.10.0-alpha.0/origin.repo > /etc/yum.repos.d/origin-master.repo; yum update -y"
```
Comment (moolitayer) [inline deployer.sh]: Looks good, but we do need a way to pull the latest unreleased 3.10 @sdodson any ideas?

Comment (sdodson) [reply deployer.sh]: There should be https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.10/origin.repo  but there isn't checking on that.

Comment (sdodson) [reply deployer.sh]: Ok, that .repo file now exists. I'd use that.

**Review by moolitayer — APPROVED** [review]
LGTM :+1:

File: deployer.sh (hunk 1 above)
Comment (moolitayer) [reply deployer.sh]: Thanks @sdodson!

**Comment by elad661** [comment]
I'm not merging because It still doesn't work... the installer is looking for an image tag that doesn't exist in dockerhub, so deployment fails

**Comment by elad661** [comment]
merging, this works now

**Pull request merged**